app/gui/view/component-browser/component-group/src/wide.rs

```diff
@@ -0,0 +1,432 @@
+//! A multi-column [Component Group] without header.
+//!
+//! The widget is defined by the [`View`].
+//!
+//! To learn more about component groups, see the [Component Browser Design
+//! Document](https://github.com/enso-org/design/blob/e6cffec2dd6d16688164f04a4ef0d9dff998c3e7/epics/component-browser/design.md).
+//!
+//! [Component Group]: crate::component_group::View
+
+use ensogl_core::application::traits::*;
+use ensogl_core::display::shape::*;
+use ensogl_core::prelude::*;
+
+use crate::EntryId;
+
+use enso_frp as frp;
+use ensogl_core::application::shortcut::Shortcut;
+use ensogl_core::application::Application;
+use ensogl_core::data::color::Rgba;
+use ensogl_core::display;
+use ensogl_gui_component::component;
+use ensogl_label::Label;
+use ensogl_list_view as list_view;
+use list_view::entry::AnyModelProvider;
+
+
+// =================
+// === Constants ===
+// =================
+
+const NO_ITEMS_LABEL_TEXT: &str = "No local variables";
+const ENTRY_HEIGHT: f32 = list_view::entry::HEIGHT;
+const MINIMAL_HEIGHT: f32 = ENTRY_HEIGHT;
+const COLUMNS: usize = 3;
```

Comment:
Does not make any sense without explanation and docs. IMO this should be a parameter to the widget. Or at least should have docs.

Reply:
I made this a const generic for most types in the module, because I thought it is fun and perfectly follows the purpose of this constant. If you find the resulting code too complicated - please ping me. 
Making this a dynamic parameter (say, an FRP input) requires a lot more effort. (we need to reshuffle entries between lists and also create/destroy them dynamically)

Reply:
Again, my comment is invalid,  apologize for it. I was 100% sure this is a generic, 3-column layout of things, not a component that is component-browser specific only. I overlooked the name of the module. So, let's keep it as is!